docs/patterns/contextual-menu.md

```diff
@@ -57,9 +57,9 @@ To import just this component into your project, copy the snippet below and incl
 @import 'patterns_contextual-menu';
 @include vf-p-contextual-menu;
 
-// when using indicator variant you need to include icons as well
+// when using indicator variant you need to include contextual menu icon as well
```

Comment:
'Indicator variant' sounds a bit confusing - we then refer to it as "contextual menu icon" - shouldn't these use the same term in both places?

Something like "If using the variant with an icon, you need to include that separately" - but maybe a native speaker like @deadlight could help?

Reply:
Good point. And I just noticed you actually need `p-button` as well, so I updated that.